specification/dartLangSpec.tex

```diff
@@ -13800,15 +13801,17 @@ \subsubsection{Cascades}
 }
 
 
-\subsubsection{Super Invocation}
-\LMLabel{superInvocation}
+\subsubsection{Superinvocations}
+\LMLabel{superInvocations}
 
-% Conditional super invocation is meaningless: \THIS{} is not null.
+% Conditional superinvocations are meaningless: \THIS{} is not null.
```

Comment:
Weeeeeell, technically ... `this` can be `null` for members of `Null`. (But it's perfectly fine to ignore that, it only affects, well, me.)

Reply:
;-)